doc/devdocs/readme.md

```diff
@@ -80,9 +80,24 @@ modify --installpath "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\%targetFo
 - The PowerToys binaries will be in your repo under `x64\Release`.
 - If you want to copy the `PowerToys.exe` binary to a different location, you'll also need to copy the `modules` and the `svgs` folders.
 
+### Temporary build-fix for Toolset v142 (Visual Studio 2019 Version 16.8)
+
+When you use the Toolset v142 (came  typical with Visual Studio 16.8) and you cannot build the C++ project `common-md-flag` you can try one of the follow solutions.
+
+Add the two compiler switches `/await` and `/Zc:twoPhase-` to the `common-md-flag` project.
```

Comment:
based on two options, this is the preferred solution

Reply:
does adding this break prior versions of VS?

Reply:
I haven't test it. I always use the newest preview from Visual Studio.

I don't know why this is working, but the sound is IMHO not good when we deactivate a check. 
But I have try to find a another way without change the SDK version

Reply:
guidance should always work against RTM, not preview.

Reply:
IMHO I think we can't say "You what to work on PT, okay you must use the none preview version of VS"

Based on the last developer survey from Microsoft, 
https://devblogs.microsoft.com/dotnet/repo-experience-survey-results/
we should improve the build documentation to reach a larger number of people

Reply:
I said it must work against RTM version of VS. You can't do a preview only change that breaks RTM.

Not just from a dev/community stance but also our build farm & CI all use RTM versions :)